spec/setup_test_database.rb

```diff
@@ -0,0 +1,6 @@
+require 'pg'
+
+def setup_test_database
+  connection = PG.connect(dbname: 'chitter_test')
+  connection.exec("TRUNCATE messages;")
```

Comment:
useful tip i found last week, if you do "TRUNCATE messages, RESTART IDENTITY;" it resets the id counter as well. as currently everytime you reset it, while it wipes the table it still keeps counting from where you left off, so after a few rspecs youll be on silly numbers in the ID column. restart identity resets it to 1 each time. probably not essential, but quite nice I thought!